build.gradle.kts

```diff
@@ -5,323 +5,325 @@ import kotlin.collections.ArrayList
+              + "}",
+          pListFilename
+      ))
+      val processBuilder2 = ProcessBuilder()
+      processBuilder2.command(parameters2)
+      val process2 = processBuilder2.start()
+      if (process2.waitFor() != 0) {
+        throw GradleException("Error while patching Info.plist")
+      }
+      val parameters3 = ArrayList<String>(Arrays.asList(
+          "mv", "$buildDir/dist/Info.plist", pListFilename
+      ))
+      val processBuilder3 = ProcessBuilder()
+      processBuilder3.command(parameters3)
+      val process3 = processBuilder3.start()
+      if (process3.waitFor() != 0) {
+        throw GradleException("Error while moving Info.plist into app")
       }
-   }
+      val parameters4 = ArrayList<String>(Arrays.asList(
+          "codesign", "--remove-signature", appDirname
+      ))
+      val processBuilder4 = ProcessBuilder()
+      processBuilder4.command(parameters4)
+      val process4 = processBuilder4.start()
+      if (process4.waitFor() != 0) {
+        throw GradleException("Error while executing codesign --remove-signature")
+      }
+    }
+  }
 }
 
 tasks.register("createDmg") {
-   group = "build"
-   description = "Makes the Mac dmg package"
-   dependsOn("createApp")
-   inputs.dir(ext.get("appDirname") as String)
-   outputs.file(ext.get("dmgFilename") as String)
-   doLast {
-      if (OperatingSystem.current().isMacOsX) {
-         val appName = ext.get("uppercaseProjectName") as String
-         val parameters1 = ArrayList<String>(Arrays.asList(
-            ext.get("jpackagecmd") as String,
-            "--type", "dmg",
-            "--app-image", "$buildDir" + File.separator + "dist" + File.separator +  appName + ".app",
-            "--name", appName,
-            "--app-version", project.version as String,
-            "--dest", "$buildDir/dist"
-         ))
-         val processBuilder1 = ProcessBuilder()
-         processBuilder1.command(parameters1)
-         val process1 = processBuilder1.start()
-         if (process1.waitFor() != 0) {
-            throw GradleException("Error while creating dmg package")
-         }
+  group = "build"
+  description = "Makes the Mac dmg package"
+  dependsOn("createApp")
+  inputs.dir(ext.get("appDirname") as String)
+  outputs.file(ext.get("dmgFilename") as String)
+  doLast {
+    if (OperatingSystem.current().isMacOsX) {
+      val parameters1 = ArrayList<String>(Arrays.asList(
+          ext.get("jPackageCmd") as String,
+          "--type", "dmg",
+          "--app-image", ext.get("appDirname") as String,
+          "--name", project.name as String,
+          "--app-version", project.version as String,
+          "--dest", "$buildDir/dist"
+      ))
+      val processBuilder1 = ProcessBuilder()
+      processBuilder1.command(parameters1)
+      val process1 = processBuilder1.start()
+      if (process1.waitFor() != 0) {
+        throw GradleException("Error while creating dmg package")
```

Comment:
Shouldn't `dmg` be spelled `DMG` (all caps)?

Reply:
That is the preferred form in English (since it is an acronym for Disk iMaGe). I was really thinking "the package ending in .dmg". I may change those to say "the .dmg package" and change the others similarly.

Reply:
I meant that it looks like commonly used form to capitalize file format acronym when referring. I.e.  you can say i.e. "Upload PNG file" or "No EXE allowed." and no further explanation is needed. So by that's why I believe macOS users will understand ""Error while creating DMG package." w/o problems as it follows the same pattern.

Reply:
I have an odd reaction to the use of uppercase extensions. In the 1980s the standard (on Unix systems) was to refer to them as, for example, .jpeg. The introduction of uppercase references came with MS-DOS. MS-DOS did not allow lowercase characters in file names so it was, in fact, EXE. But they also converted lots of Unix extensions to uppercase (and also restricted them to 3 characters, hence JPG). After nearly 40 years I still get an internal cringe when I see uppercase extensions. So I prefer .dmg to DMG when referring to the file extension.

Reply:
But I am not talking about file name extension! This should be lower-cased, no doubts. I am talking about referring to file format in ordinary message (maybe mine "in normal texts" wasn't clear, sorry).